File: src/main/java/com/greentree/telegram/queue/model/Queue.java
package com.greentree.telegram.queue.model;

import jakarta.persistence.Entity;
import jakarta.persistence.GeneratedValue;
import jakarta.persistence.Id;
import lombok.AllArgsConstructor;
import lombok.Builder;
import lombok.Data;
import lombok.NoArgsConstructor;

@Entity(name = "Queue")
@Data
@Builder
@AllArgsConstructor
@NoArgsConstructor
public class Queue {

    @Id
    @GeneratedValue
    private long id;

}

File: src/main/java/com/greentree/telegram/queue/model/Queue.kt
package com.greentree.telegram.queue.model

import jakarta.persistence.Column
import jakarta.persistence.Entity
import jakarta.persistence.GeneratedValue
import jakarta.persistence.Id
import lombok.Data

@Entity(name = "Queue")
@Data
data class Queue @JvmOverloads constructor(
	@Column
	var name: String = "",
	@Id
	@GeneratedValue
	var id: Long? = null,
)
